sample/src/samples/datepicker/validation.js

```diff
@@ -0,0 +1,45 @@
+import { inject, NewInstance } from 'aurelia-framework';
+import { ValidationController, ValidationRules } from 'aurelia-validation';
+import { MaterializeFormValidationRenderer } from 'aurelia-materialize-bridge';
+
+@inject(NewInstance.of(ValidationController))
+export class Validation {
+  date = '';
+  date2 = '';
+  showErrortext = false;
+
+  controller = null;
+
+  rules = ValidationRules
+    .ensure('date')
+      .displayName('Date')
+      .required()
+    .ensure('date2')
+      .displayName('Show or hide error text date')
+      .required()
+    .on(this)
+    .rules;
+
+  constructor(controller: ValidationController) {
+    this.controller = controller;
+    this.controller.addRenderer(new MaterializeFormValidationRenderer());
+  }
+
+  reset() {
+    this.selectedValue = '';
```

Comment:
I guess this should be `date`.. 😃 
Also, if you set it to an empty string, it will contain the current date. Set it to `null` and it works.